packages/commonwealth/client/scripts/views/components/Comments/ArchiveMsg/ArchiveMsg.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { CWIcon } from 'views/components/component_kit/cw_icons/cw_icon';
+import moment from 'moment';
+
+type ArchiveMsgProps = {
+  archivedAt: moment.Moment | null;
+}
+
+export const ArchiveMsg = ({
+  archivedAt,
+}: ArchiveMsgProps) => {
+  return (
+    <div className="archive-msg-container">
+      <div className="archive-msg">
+      <CWIcon
+        iconName="archiveTrayFilled"
+        iconSize="small"
+      />
+      {`This thread was archived on ${moment(archivedAt).format('MM/DD/YYYY')},
+      meaning it can no longer be edited or commented on.`}
+      </div>
+    </div>
+  )
+}
```

Comment:
prettier: file is not formatted correctly

Reply:
Done!